test/Redistribution.test.ts

```diff
@@ -873,6 +883,10 @@ describe('Redistribution', function () {
           priceOracle = await ethers.getContract('PriceOracle', deployer);
           await priceOracle.unPause(); // TODO: remove when price oracle is not paused by default.
 
+          // Set price base
+          const priceBase = await priceOracle.priceBase();
+          priceBaseString = priceBase.toString();
```

Comment:
you cast it to int at each occurrence, why don't you just cast it here to int instead?

Reply:
was modeled after
 let minPriceString: string;
 
 as it derived from there as a new separate value. That needed a string for setPrice method, but this one doesn't and you are right, it shouldn't be cast back and forth. I will improve that part. thnx